weblate/addons/models.py

```diff
@@ -81,15 +92,23 @@ def save(
         self, force_insert=False, force_update=False, using=None, update_fields=None
     ):
         cls = self.addon_class
-        self.project_scope = cls.project_scope
         self.repo_scope = cls.repo_scope
+        self.project_scope = cls.project_scope
+
+        if self.project_scope:
+            self.project = self.component.project
+            self.component = None
```

Comment:
Do not set non-existing attribute on a class:

```suggestion
        original_component = None
        if cls.project_scope:
            original_component = self.component
            self.project = self.component.project
            self.component = None
```